### DISCUSSION TITLE
Max Fan Speeds

### DISCUSSION THREAD

**Discussion by seerge** [seerge/g-helper#1279] [General]
Hello fellow community,

### Option to "calibrate" Y-Axis in fan editor
In reality custom fan curve is being set in % values, not in RPM. Y-Axis in fan editor is for a indication only.
By default app would assume 100% = 5600RPM and update it to higher values if it sees your fan actually going faster.

Now you can manually click Calibrate button in Fans+Power, that would run a quick process of calibration when app would try to set maximum fan speed to all fans, measure them and update Axis accordingly

**Important notice** : actual fan behavior is not affected anyhow! It's just to show more accurate RPMs in the editor :)

-------------------

I can also pre-enter those values , so new users won't have to do that :)

Can you run Calibration in the app and post your measurements here along with model name ?

Example

**G14 2022 GA402RK** : CPU : 5500, GPU : 5600

Thanks

---

Collected settings so far with pre-entered values 

```
            if (AppConfig.ContainsModel("GA401I")) return new int[3] { 78, 76, DEFAULT_FAN_MAX };
            if (AppConfig.ContainsModel("GA401")) return new int[3] { 71, 73, DEFAULT_FAN_MAX };
            if (AppConfig.ContainsModel("GA402")) return new int[3] { 55, 56, DEFAULT_FAN_MAX };

            if (AppConfig.ContainsModel("G513R")) return new int[3] { 58, 60, DEFAULT_FAN_MAX };
            if (AppConfig.ContainsModel("G513Q")) return new int[3] { 69, 69, DEFAULT_FAN_MAX };
            if (AppConfig.ContainsModel("GA503")) return new int[3] { 64, 64, DEFAULT_FAN_MAX };

            if (AppConfig.ContainsModel("GU603")) return new int[3] { 62, 64, DEFAULT_FAN_MAX };

            if (AppConfig.ContainsModel("FA507R")) return new int[3] { 63, 57, DEFAULT_FAN_MAX };
            if (AppConfig.ContainsModel("FA507X")) return new int[3] { 63, 68, DEFAULT_FAN_MAX };

            if (AppConfig.ContainsModel("GX650")) return new int[3] { 62, 62, DEFAULT_FAN_MAX };

            if (AppConfig.ContainsModel("G732")) return new int[3] { 61, 60, DEFAULT_FAN_MAX };
            if (AppConfig.ContainsModel("G713")) return new int[3] { 56, 60, DEFAULT_FAN_MAX };

            if (AppConfig.ContainsModel("Z301")) return new int[3] { 72, 64, DEFAULT_FAN_MAX };

            if (AppConfig.ContainsModel("GV601")) return new int[3] { 78, 59, 85 };

            return new int[3] { DEFAULT_FAN_MAX, DEFAULT_FAN_MAX, DEFAULT_FAN_MAX };

```

**Comment by Aragonii** [comment]
G14 2020 GA401IV (AMD 4900HS/Nvidia 2060)         CPU:8000    GPU:7900

**Comment by DaddyK1tt3n** [comment]
G513QM 2021 : CPU : 6900 GPU : 6900

**Comment by woossa1987** [comment]
Z13 2023: 
CPU:7200
GPU:6400

**Reply by rainbowflesh** [reply]
The GV301VV got 7400 cpu fan speed.

**Comment by ZiKion** [comment]
G14 GA401QM （5900HS/RTX 3060）CPU：7100 GPU：7300

**Comment by Zoe020512** [comment]
tuf 2021 FA506QM （5800H/3060) CPU:5700 GPU:5700

**Comment by TheBestPessimist** [comment]
Asus ROG Flow X16 GV601VI - 2023 - 4070GPU 
Final max after multiple measurements:

CPU: `7800`
GPU: `5900`
Mid Fan: `8500`

---
Measurements:

2. `Measured - CPU: 7800, GPU: 5900, Mid: 6900`
3. `Measured - CPU: 7800, GPU: 5900, Mid: 7200`
4. `Measured - CPU: 7900, GPU: 5900, Mid: 7200`
5. `Measured - CPU: 7600, GPU: 5900, Mid: 7000`

Remarks:

1. Mid fan is not shown in the calibration screen the first time i run calibrate, however it is shown the second time i run calibrate. The third time it did not appear either.
![image](https://github.com/seerge/g-helper/assets/4482210/0ca185ed-ceb9-4902-b453-c3b21de80973)
1.1. It seems that the mid fan is simply not displayed until i click to copy and/or the measurement ends.

2. The first run, i saw mid fan reach 8500, the second run GH showed mid fan reach only 6900, and i pasted more measurements above.
2.1. The mid fan measurement never reached `8500` in the measurement screen, instead i saw it reach `8500` in the GH main window. Which is the correct value?

3. The "max" values are taken from GH main window, not from the "Calibration".

**Reply by seerge** [reply]
I have added "mid" fan output into measurement steps :)

As for your mid fan behaving differently - i don't know why (BIOS doesn't want to go high on second run? or has some tricky algorithms inside?)

**Reply by NJinsei** [reply]
I tried calibrating the fan curve on my Flow X16 and noticed that the calibration had only spun the midfan to around 76% which might explain why lower values are reported in some measurements.

**Reply by seerge** [reply]
It sets fan curve to 100% for all fans (including mid), purpose of calibration is actually to know what is max RPM it can go up to.

This is because you set fans in %, and read sensors in RPM with asus :) So after calibrating fans should go up to 100% (just 100% now will mean different RPM)

**Reply by NJinsei** [reply]
Ah I see that makes sense, thank you for the clarification.

**Comment by Li-Qirun** [comment]
ROG Zephyus M16 GU603HM 2021 (11800H/3060) 
CPU: 6700
GPU: 6700

**Reply by bmyjacks** [reply]
In my GU603HM it could not reach so high RPM.
CPU: 6200
GPU: 6400

**Comment by spuggle** [comment]
TUF A15 FA507RE 2022 (6800H/3050Ti)
CPU: `6300`
GPU: `5700`

**Comment by Twel12** [comment]
Zephyrus G15 GA503QM 2021(5900HS/3060)
CPU : 6400
GPU: 6400

**Comment by utchka** [comment]
Are you sure calibration is working properly? Mine says 7300 rpm for both cpu/gpu, but I'm sure it's a lot less than that. 
When I had armoury crate installed the fans never got over 5000 rpm., not even in max performance mode under heavy cpu testing.
Also, at 2000 rpm G-Helper says that the fan is working at 36%, so 7300 rpm would be about 130%.
What am I getting wrong?
![fans-rpm](https://github.com/seerge/g-helper/assets/139227466/4bda1a19-7df4-4347-b203-4ae6583c6655)

**Reply by seerge** [reply]
What's your definition of "properly" ? It just sets fan curve to 100% and looks at FAN RPM being reported by bios, and remembers them.

**Comment by enprezzo** [comment]
G513RM (6800h/3060): 5800RPM CPU, 6000RPM GPU

**Comment by bmyjacks** [comment]
Zephyrus DUO 16 2023 GX650PY (7945HX/4090)
CPU: 6200
GPU: 6200

**Comment by BuharTohru** [comment]
G14 2023 GA402XZ（7940HS/RTX 4080) : CPU : 5500, GPU : 5600

**Comment by squee666** [comment]
Zephyrus DUO 16 2023 GX650PZ (7945HX/4080)
CPU: 6200
GPU: 6200

**Comment by 123dqwkhjk** [comment]
g15 2022 ga503rm
CPU:6600rpm
GPU:6400rpm

**Comment by MDowski** [comment]
2020 **G14 GA401U** (4900 HS + 1660 Ti) - CPU: 7800, GPU 7600-7700 (depends on the run).

I'm certain that MyASUS spun it to 7900 RPM (according to Armoury Crate measures), but maybe dust on the blades shaves off some RPM? I haven't cleaned it in a while. Or simply AC was incorrect.

What is more, [factory settings button is now too narrow in a Polish translation](https://i.imgur.com/iImuts6.png).

**Reply by seerge** [reply]
Fan testing command (from myasus) can actually spin fans faster than any custom fan curve. 

But whole purpose of this "calibration" is actually to establish maximum speed that fans can go from curve being set to 100% (so after that app can make Y-axis more accurate with RPMs) ;)

**Comment by manciuszz** [comment]
ROG Strix 17 G732LXS.308
CPU: 6100 RPM
GPU: 6000 RPM

**Reply by Viper3000** [reply]
My STRIX SSCAR 17 G732LWS  

CPU MAX 9100
GPU 8000

Only in Windows beacuse ASuS ....

in game only CPU/GPU 6400 MAX

**Comment by Metalogic999** [comment]
2023 Rog Zephyrus G14 GA402XZ (7940HS CPU + RTX 4080 GPU)

CPU: 5500 rpm
GPU: 5800 rpm

**Comment by GlazedBelmont** [comment]
2022 G14 - GA402RJ 
CPU: 5600 RPM
GPU: 5700 RPM

**Comment by Maneeha1234SL** [comment]
GA503RM 2022 
Ryzen 7 6800hs
RTX 3060

CPU: 6400 RPM
GPU: 6400 RPM

It would be great if cpu SPL/SPPT/FPPT power settings and GPU base clock offset/Memory clock offset/Dynamic boost/thermal target/Gpu Power saver can be set to suit each computer.

<img width="821" alt="image" src="https://github.com/seerge/g-helper/assets/83775004/62752102-1655-4aea-9de5-bdfb578d0ee7">
<img width="818" alt="image" src="https://github.com/seerge/g-helper/assets/83775004/70db1c4e-cbd3-492c-91d3-72a583eb04b8">

**Reply by seerge** [reply]
Those settings have no effect when unchecked, and when checked they still have own hard limits on BIOS level.

**Reply by Maneeha1234SL** [reply]
@seerge 

Important Notice

The forthcoming BIOS update, version 318, for our Asus Zephyrus GA503RM laptops is set to address the Fan Curve Swap Issue identified in BIOS 317.

In BIOS 318, there have been adjustments to the fan speed when thermal demands are at their peak:

CPU: 6700 RPM
GPU: 6600 RPM

Currently, the maximum fan speeds in G Helper are set at 6400 RPM for the GPU and 6600 RPM for the CPU. These values will need to be updated accordingly.
<img width="1280" alt="Screenshot 2024-03-10 200334" src="https://github.com/seerge/g-helper/assets/83775004/f6fcd868-ba15-4f0c-811b-6f6868a43387">
<img width="1187" alt="image" src="https://github.com/seerge/g-helper/assets/83775004/f3ff683d-2092-4ea8-9081-c562e8c09567">

**Reply by seerge** [reply]
@Maneeha1234SL app would automatically update maximum values on Y-axis as soon as it spots higher values :)

**Comment by Constrat** [comment]
Strix G17 G713RW
`CPU: 5500RPM`
`GPU: 6000RPM`

**Comment by chonkycarlos** [comment]
I7-9750H
2060
M15 2019 GU502GV : CPU : 7200, GPU : 7200 (I had to set the fan speed max before calibrating)

**Comment by eggseenose** [comment]
Flow X16 (GV601RM)
-CPU: 7700
-GPU: 5800
-Mid: 6100

**Comment by TheFlano23** [comment]
G15 G513QC-HN008T 2021:
CPU fan speed: 5200RPM
GPU fan speed: 5400RPM

**Comment by yudapedia** [comment]
2022 G14 GA402RK
CPU: 6000
GPU: 6000

![image](https://github.com/seerge/g-helper/assets/61176703/15419c78-d0f5-48c4-894a-fcdd579b704b)

**Comment by Kikimahe** [comment]
TUF 2022 FX707ZM

CPU : 6200
GPU : 7000

**Comment by marcelomijas** [comment]
ROG Zephyrus G14 GA401IV (AMD Ryzen 7 4800HS, NVIDIA RTX 2060)

CPU: 8000, GPU: 8000

![imagen](https://github.com/seerge/g-helper/assets/62946623/95b750c1-8df5-4a57-8448-d353d1d2d1cd)

_Airplane mode on my laptop takes a new meaning_

**Reply by GusssCM** [reply]
Got #### Johnny! I have the same laptop 2020 and I got CPU: 8000 GPU: 7900. An oldie but goldie right?

**Comment by ghost** [comment]
Does not apply to laptops which do not support custom fan curves, wonder if hiding it on these laptops is better? (if it is even possible to predetermine this)

**Reply by seerge** [reply]
That's a good point :) There is no definitive way to determine if fan curves are supported or not.

But since calibration is extremely optional feature, I can hide it for devices where BIOS doesn't report a fan curve at all

I have uploaded 0.122 with this "hotfixed" addition https://github.com/seerge/g-helper/releases/download/v0.122/GHelper.zip

**Reply by syh599** [reply]
Hi, I think this hotfix has mistakenly hidden the button for some devices that actually support the feature. On my G15 GA503RM the calibration button cannot be seen starting from 0.122

**Comment by b66ll** [comment]
G513QM 2021

ryzen 7 5800h rtx 3060 ( CPU: 6600, GPU: 6600)


![لقطة الشاشة 2023-09-12 040325](https://github.com/seerge/g-helper/assets/144750283/518b7033-5b0c-4919-a32b-e5513507209d)


GHelper 0.122

**Reply by b66ll** [reply]
seerge ....

I have a problem with a laptop that supports a slow or fast processor in the Armory Crate
There is also a fast graphics card, like the picture on the left side
But in GHelper it is not supported

![لقطة الشاشة 2023-09-12 042141](https://github.com/seerge/g-helper/assets/144750283/4f581da6-228f-490f-a6db-4b2431990091)


May God bless your life

**Reply by seerge** [reply]
Hello, on screenshot both sliders are just combined into one :) effect is same

**Reply by b66ll** [reply]
You are really amazing person . ok thx

**Comment by Riccorbypro** [comment]
ROG Zephyrus DUO 16 (2022) GX650RW (Ryzen 9 6900HX, RTX 3070Ti):
CPU: 6200
GPU: 6200

**Comment by patrpok** [comment]
**G14 2023 GA402XV** : CPU : 5500, GPU : 5800

**Comment by G531GV** [comment]
Dear all
ASUS ROG Strix G531GV i7-9750H/RTX2060 : CPU : 6100, GPU : 6100
By enabling fan curve, showing ASUS BIOS REJECTED FAN CURVE
Bios 306
![Screenshot 2023-09-12 184745](https://github.com/seerge/g-helper/assets/129619875/d8072f8d-6ef7-4f06-9d3d-ca7956399c13)

**Comment by akshat-aqui** [comment]
G14 GA402RJ 2022 (6800HS + 6700S)
CPU: 5500
GPU: 5400

**Comment by IanFluidd** [comment]
UPDATED as my previous results were based on one test, I have now performed multiple and taken the highest value.

G15 2021 GA503QR: 
CPU: 6400
GPU: 6400

**Reply by ke1ne** [reply]
The same for G15 2021 GA503QR:
CPU: 6400
GPU: 6400

**Comment by NJinsei** [comment]
ROG Flow X16 GV601RW:
CPU: 7400
GPU: 5600
Mid Fan: 6400

**Comment by mezmer420** [comment]
**G14 2022 GA402RK**
CPU: 6000
GPU: 6000

**Comment by freebird920** [comment]
ROG FLOW X13(2022) GV301RA
CPU 9000
GPU 6800

**Comment by Matyyy** [comment]
Asus Rog Strix G533ZW-AS94 (2022)
CPU: 6800
GPU: 6700

![2023-09-12 21_51_01-](https://github.com/seerge/g-helper/assets/23487973/7208c12c-88aa-4c31-b605-f6423411e27b)

**Comment by adrian-aaen** [comment]
ROG FLOW X13(2021) GV301QH
CPU 8800
GPU 3500

Doesn't look like XGM rpm was measured though.

**Comment by freebird920** [comment]
cpu -100
gpu 6800
<img width="834" alt="스크린샷 2023-09-15 010802" src="https://github.com/seerge/g-helper/assets/53999927/211fc9ad-74dd-4736-af1f-db2becef1aca">
 
something wrong??

**Reply by seerge** [reply]
@freebird920 run again? :)

**Reply by Viper3000** [reply]
YES my scar 17 shows same RPM after 15s-30s after calibration.

**Reply by seerge** [reply]
@freebird920 @Viper3000 it could be that if fan reports more than 9000RPM (is it true tho?) app would discard that as nonsense 

Try this build, I have raised limit of "no-nonsense" RPM to 9500
[GHelper.zip](https://github.com/seerge/g-helper/files/12611767/GHelper.zip)

**Reply by freebird920** [reply]
@seerge I tried that build.  and this is the result. 
<img width="831" alt="스크린샷 2023-09-15 071258" src="https://github.com/seerge/g-helper/assets/53999927/a750c2a8-d41f-4e88-a2e8-75ffee640ca3">

**Comment by Viper3000** [comment]
I think that now is ok see added picture [https://ulozto.sk/file/3nsfMRssyPb2/14-9-png#!ZJWvZmR2ZwH2AJR5ZmqyBTSyBGqzAGWvHUyloUVkZIHlJJWxAD==](url)

sorry im not sure how I can add picture on github

**Reply by seerge** [reply]
Ok, but is 9000 RPM that your sensor reports is actually realistic ? Or it's some bug in BIOS? (i.e. can you achieve same RPM via armoury manual mode?) I just can't imagine how it sounds even ... :D

**Reply by Viper3000** [reply]
This is my last picture from Asus Fuc-k crate system [https://ulozto.sk/fm/detail/qsHUMe9Y32KO/8TTn1jOWsgrq](url)
I have  a different problem. And that is that when I turn on the game and play the game, the fan speed does not want to exceed 6500rpm. And then the laptop is 98-103 degrees and comes to underclocking.

**Reply by seerge** [reply]
I think that's your "real" maximum RPM, or some kind of built in protection in bios (as it controls fans) to protect them from wearing off

**Comment by syh599** [comment]
G15 2022 GA503RM : CPU : 6600, GPU : 6200

**Comment by AestenD** [comment]
G14 2022 GA402RJ : CPU : 6000, GPU : 6000

**Comment by lucidnx** [comment]
GA402RK-L8150W
CPU: 5600rpm
GPU: 5700rpm

**Comment by goranovs** [comment]
Rog Zephyrus M16 (GU603HM)

With **MyAsus** Fan Test:
CPU: 9000 rpm
GPU: 8600 rpm

Turbo mode in game:
CPU: 7800 rpm
GPU: 7600 rpm 

Unfortunately, applying any custom curves limits both fans to 6600 - 6700 rpms

**Comment by Viper3000** [comment]
I have big problem on my Asus ...Please see TimeSpy results on my notebook when It was NEW februar 2021/ and now september 2023

Even though max RPM on my notebook is CPU9000/GPU8000

https://www.3dmark.com/spy/18387432    (8368points) - Turbo Asus Crate 
https://www.3dmark.com/3dm/100062685   (6382points) - Turbo in Asus Crate and also G-Helper   beacause max RPM in Asus Crate and g-helper during test is 6500 RPM  

Im not sure where is problem - but my notebook it is not as powerful as it was

**Reply by lucidnx** [reply]
I guess this is not place to discuss performance...

**Comment by martink777** [comment]
G15 GA503QS (2021 - Ryzen 9 5900HS/RTX 3080)
CPU: 6400
GPU: 6400

**Comment by Kikimahe** [comment]
Hi

I did a clean install of WINDOWS 11.

1) For more information, to run GHELPER, Windows asked me to install Windowsdesktop runtime 7.0.

2) GHELPER works correctly but the calibration button has disappeared!?
(v0.124, 0.123 and 0.122)

![Capture d’écran 2023-09-19 131813](https://github.com/seerge/g-helper/assets/142881210/9a000888-3c79-4f48-aff1-b6790ea57cc7)

**Reply by seerge** [reply]
@Kikimahe i have removed calibration button for models that don't return default fan curves (as there is no way to check if they support them or not before actually trying to "apply" them).

App will still calibrate fan speeds based on what it sees from a sensor (i.e. if it will see fan spinning faster than default 5800RPM it will adjust axis next time you open fans+power)

**Reply by Kikimahe** [reply]
Yes Seerge, you already answered me in another thread. ;-)

The automatic calibration works very well.
By default it was at 5800 on the 2 curves and now they have gone to CPU 6400 and GPU 7000.

Your utility is amazing and I think it's the best thing installed on my laptop. I love watching it work. :-)
![Capture d'écran 2023-09-28 132853](https://github.com/seerge/g-helper/assets/142881210/aa1530fa-913c-4e7d-b1b1-128c27b7ffe0)

**Comment by IceStormNG** [comment]
M16 2023 GU604VZ

CPU: 7000
GPU: 7200
Mid: 5700, though this seems quite wrong. The mid fan should easily go up to 7000 or so. AC reports 7200 or more for the mid fan... except... if they changed that in the latest bios update.

**Comment by caizefeng** [comment]
G14 2023 GA402XV（7940HS w/ RTX 4060, BIOS 313)
CPU: 5500
GPU: 5800

**Comment by shinikamy** [comment]
g733qs here. I know this is fan max speed discussion but can you also allow for fan curve to work when cpu and gpu are under 20*C ? i'm one of the guys that has the issue with freezing device and every time fan drops under 2000 rpms the laptop freezes. at night when laptop doesn't do much....it has a moment where it goes to 0 rpms and inevitably it stops the laptop....it might be a bios issue of some sorts, a lot of ppl have it but nothing from asus or AMD yet....the only work around is to keep fans above 2000 rprms. ty

**Reply by seerge** [reply]
@shinikamy hello. Everything that is possible, is already available in the app. You can try to set a custom fan curve where first dot would be at 20C / 2000RPM and preferably in Turbo or Balanced mode :) rest is up to BIOS in your laptop, on how to treat that and how to control fans.

**Comment by navhawk80** [comment]
G14 2022 GA402RJ : CPU : 6000, GPU : 6000
PS: Ryzen 7 6800HS / RX6700S, BIOS 319

**Comment by chockokruasan** [comment]
Rog Zephyrus duo 16 GX650PZ
CPU: 6200rpm
GPU: 6200rpm

Somebody know why ASUS locked MAX fan speeds for TURBO mode at 5400rpm? I always getting 100-102 degrees CPUand 87 degrees GPU on TURBO factory mode. Is it OK for laptop health to apply max rpm 6200?

**Reply by Arvilsy** [reply]
yeah that what im wondering too. mine on myasus app and turbo mode runs at 5800 max. but with calibration runs at 6100, idk if its gonna make the fans worse or broken overtime

**Comment by Raghav-Kochhar** [comment]
2021 TUF Gaming F15, FX506HM, i5-11400H & RTX 3060:
- CPU: 6000
- GPU: 7000

**Comment by avaensis** [comment]
F16 2024 FX607JI  (13980HX/4070): CPU : 7400, GPU : 7200

**Reply by avaensis** [reply]
![Untitled](https://github.com/seerge/g-helper/assets/167683291/2d6b09cb-c45d-4f89-b1bf-a6e30224ff0e)

Not sure if this is a bug, as you can see in the picture cpu is at 56 degrees and according to the graph the fan should run at less than 2800rpm, but is reported to be running at 3000rpm

**Reply by seerge** [reply]
@avaensis you don't have a custom fan curve applied, so BIOS uses some own fan curve here. So it's normal.

Also even if you would set a custom fan curve, BIOS will still behave as it was programmed by asus. So it may not follow it super precisely.

**Comment by Arkanad** [comment]
Hello, maybe someone could help? Because i don't think that is normal RPM... 
![Знімок екрана 2024-07-09 151835](https://github.com/seerge/g-helper/assets/110057821/17a4a990-04de-4be5-b3d4-ff841f1194da)

**Reply by seerge** [reply]
@Arkanad hello :) it looks like sensor is reporting something "random".  What does Armoury show to you in this situation ?

**Reply by Arkanad** [reply]
> @Arkanad hello :) it looks like sensor is reporting something "random". What does Armoury show to you in this situation ?

![image](https://github.com/seerge/g-helper/assets/110057821/0bf027bf-38d8-4489-afea-83996cdb346c)
Something like this. i am also facing an issue with cpu high frequency and cpu fan working at 100% in any mode, right after turning on the laptop, 
![image](https://github.com/seerge/g-helper/assets/110057821/df6f8cd5-fb7f-4765-ba84-f445cf794dba)
I tried everything i could to fix it, but no luck, few months already. G-helper provides temperatures, but shows something "random" fan speed.
I installed it not a long time ago, had to give it a try. In the end i think something with laptop and it's seems unsolvable :D

**Reply by seerge** [reply]
@Arkanad so GPU fan always reports 0 in AC ? seems like something is wrong either with fans or with firmware.

Try to do [hardware reset](https://github.com/seerge/g-helper/wiki/Troubleshooting#hardware-reset--cmos-reset), may be that will help somehow

**Reply by Arkanad** [reply]
> @Arkanad so GPU fan always reports 0 in AC ? seems like something is wrong either with fans or with firmware.
> 
> Try to do [hardware reset](https://github.com/seerge/g-helper/wiki/Troubleshooting#hardware-reset--cmos-reset), may be that will help somehow

With gpu no problem i guess, it's just i didn't use anything to load gpu on the moment of making screenshot. That's all cpu, tried hardware reset - also no luck. But thanks for the advice!

**Reply by seerge** [reply]
@Arkanad you can also try [Experimental build with manual fan control](https://github.com/seerge/g-helper/discussions/2272) :) it won't "fix" incorrect sensors but may help keeping fans running as you want them to (in case of Custom fan curves)

**Reply by Arkanad** [reply]
> @Arkanad you can also try [Experimental build with manual fan control](https://github.com/seerge/g-helper/discussions/2272) :) it won't "fix" incorrect sensors but may help keeping funs running as you want them to (in case of Custom fan curves)

I found out that my fan controller is dead. I've send it to the service :D. Thanks for the fast answers tho!

**Comment by Arvilsy** [comment]
ASUS TUF F15 2022 FX507ZC4 DDR4 
Cpu:6100
Gpu:5700

**Comment by Mancitiss** [comment]
ASUS ROG Flow X16 GV601VV 4060 (2023)
CPU: 7700
GPU: 5900
MID: 8500-9300

after multiple tries I notice the mid fan can run higher, up to 9300, only when placing flat down like a normal laptop with 90 degrees screen (not folded, not upside down), other positions can run up to only only 8500 (with a stand to form a 30 degrees angle, or upside down), CPU and GPU is consistent at 7700 and 5900

**Comment by shivpxl** [comment]
TUF DASH Fx517ZM （i7 12650H/RTX 3060）CPU：6500 GPU：6500
